tests/test_eval_detection.py

```diff
@@ -215,8 +216,26 @@ def test_accumulate() -> None:
         cls_to_accum["VEHICLE"]
         == np.array(
             [
-                [1.0, 1.0, 1.0, 1.0, expected_ATE, expected_ASE, expected_AOE, expected_AP],
-                [1.0, 1.0, 1.0, 1.0, expected_ATE, expected_ASE, expected_AOE, expected_AP],
+                [
```

Comment:
i think it was more readable before, on a single line. If we do

```
# fmt: off

blah blah 

# fmt: on
blah blah
```
we can trick python black into keeping it as before